### PR TITLE
Simplify Font.swift

### PR DESCRIPTION
**Pull request by joshuatbrown**
In order to simplify Font.swift and remove ambiguity, this PR:

- removes the Text func
- removes the clarity, clarityBold, and clarityMedium vars for simplification
- actually just removes a lot of clarity... vars to funnel everything through a single `clarity(_:textStyle:)` func

### CODE REVIEW

File: Nos/Extensions/Font.swift [hunk 1]
```diff
@@ -93,7 +72,7 @@ func PlainText(_ localizedStringResource: LocalizedStringResource) -> SwiftUI.Te
     SwiftUI.Text(localizedStringResource)
 }
 
-func Text(_ content: any StringProtocol) -> SwiftUI.Text {
+func BrandText(_ content: any StringProtocol) -> SwiftUI.Text {
```
Comment (joshuatbrown) [inline Nos/Extensions/Font.swift:75]: I did a Refactor > Rename on this and would have expected it to properly catch all usages, but I'm not 100% sure it did. It's hard to tell what was using SwiftUI.Text and what was using this `func Text`...

**Review by martindsq — COMMENTED** [review]
All this looks good.
Given that you are removing `Text`, are we in a good position to remove PlainText as well. I think those who were using it will just become Text (the SwiftUI one)

**Comment by joshuatbrown** [comment]
> All this looks good. Given that you are removing `Text`, are we in a good position to remove PlainText as well. I think those who were using it will just become Text (the SwiftUI one)

I think we're good, as long as we like this direction in general. I'm finding 91 results when I search for PlainText, so I thought I'd wait on that until we decide for sure we want to do this. I'm happy to make those changes in this PR, though, if we're all in agreement. @mplorentz?

**Review by martindsq — DISMISSED** [review]
Fair enough. I will approve it for now so I don't block you.

**Comment by mplorentz** [comment]
Either way is good with me!

**Comment by joshuatbrown** [comment]
@mplorentz I went for it, so if you could review the update I'd appreciate it!

**Review by mplorentz — APPROVED** [review]
Looks good 👍

**Pull request merged**